src/SkyDrop.Droid/Services/AndroidHttpClientFactory.cs

```diff
@@ -1,34 +1,41 @@
 using System;
+using System.Collections.Generic;
 using System.IO;
 using System.IO.Compression;
 using System.Net;
 using System.Net.Http;
 using System.Net.Http.Headers;
 using System.Threading.Tasks;
 using Newtonsoft.Json;
+using SkyDrop.Core.Components;
+using SkyDrop.Core.DataModels;
 using SkyDrop.Core.Services;
 
 namespace SkyDrop.Droid.Services
 {
-    public class AndroidHttpClientFactory : ISkyDropHttpClientFactory
+    public class AndroidHttpClientFactory : BaseSkyDropHttpClientFactory
     {
```

Comment:
So this structure is handy. Allows you to use per platform dependency injection with an abstract base service in the Core project, to avoid code duplication in services which have similar per platform implementations.